README.md

```diff
@@ -260,6 +261,43 @@ it.
 NOTE: You must ensure that the data type (e.g. Object, Array or String) for
 the replaced value is correct.  This cannot be done automatically.
 
+Hjson Files
+----------
+
+Hjson is a syntax extension to JSON. It is intended to be used like a user interface for humans, to read and edit before passing the JSON data to the machine. That means you can use it to parse JSON files but it is not intended as a replacement.
+
+You can check Hjson's homepage at https://hjson.org to get familiar with it and you can try out it's syntax at https://hjson.org/try.html.
+
```

Comment:
It would be better if you actually had links in this paragraph. Such as:

You can see the [Hjson homepage](https://hjson.org) and [try out it's syntax](https://hjson.org/try.html).

Reply:
Ok, I was unsure about this.

Reply:
These .md files support any features of [GitHub Flavored Markdown](https://guides.github.com/features/mastering-markdown/). You're encouraged to use them. :-)